### PR TITLE
fix(exception_handler): log errors via `console.error`

### PR DESCRIPTION
**Pull request by tbosch**
This is e.g. needed as we use this to test for errors
in our examples.

### CODE REVIEW

File: modules/angular2/src/dom/html_adapter.dart [hunk 1]
```diff
@@ -9,6 +9,10 @@ class Html5LibDomAdapter implements DomAdapter {
     setRootDomAdapter(new Html5LibDomAdapter());
   }
 
+  logError(error) {
+    window.console.error(error);
```
Comment (vicb) [inline modules/angular2/src/dom/html_adapter.dart]: print()

File: modules/angular2/src/dom/parse5_adapter.cjs [hunk 2]
```diff
@@ -28,6 +28,11 @@ export class Parse5DomAdapter extends DomAdapter {
     setRootDomAdapter(new Parse5DomAdapter());
   }
 
+  logError(error) {
+    // TODO: Do proper formatting of errors with stdout here.
+    print(error);
```
Comment (vicb) [inline modules/angular2/src/dom/parse5_adapter.cjs]: is print available in cjs ?

File: modules/angular2/src/dom/html_adapter.dart (hunk 1 above)
Comment (rkirov) [inline modules/angular2/src/dom/html_adapter.dart]: won't work in server dart.

File: modules/angular2/src/dom/parse5_adapter.cjs (hunk 2 above)
Comment (rkirov) [inline modules/angular2/src/dom/parse5_adapter.cjs]: this should be `console.error`.

**Pull request merged**